SOURCES/cve-2019-1563.patch

```diff
@@ -0,0 +1,152 @@
+From ebe3348aed59dee92b3265828efd5d5522a93818 Mon Sep 17 00:00:00 2001
+From: Lucas RAVAGNIER <lucas.ravagnier@vates.tech>
+Date: Tue, 29 Oct 2024 10:34:00 +0100
+Subject: [PATCH 2/3] Fix a padding oracle in PKCS7_dataDecode and
+ CMS_decrypt_set1_pkey
+
+Backport notes: Drop CHANGES modifications
```

Comment:
Same as above. I vaguely understand that maybe there was a file named CHANGES that we don't modify, but I'm note sure, and more importantly it doesn't tell us why.

Reply:
Reworded it for more clarity